src/codeu/chat/server/Controller.java

```diff
@@ -147,6 +164,67 @@ public Conversation newConversation(Uuid id, String title, Uuid owner, Time crea
     return conversation;
   }
 
+  // Search for a user in the database and create the User object in the return.
+  @Override
+  public User searchUserInDatabase(String username, String pswd){
+    DatabaseUser tempUser = databaseUsersMap.get(username);
+    //return (tempUser != null) ? (tempUser.pswd.equals(pswd)) ? model.userById().first(tempUser.id) : null : null;
```

Comment:
It's okay to comment out parts of your code temporarily when you don't know if you still need it or not, but you should clean up your code before committing.